int16 health 
int16 modID
time stamp
